### ISSUE TITLE
Update Ben Karpinski's photo attribution link from Fall 2019 wallpapers - Follow up to 7997

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#8831]
## Description 
Follow up to https://github.com/brave/brave-browser/issues/7997

From above issue:
```
Currently, Ben Karpinski's photo attribution link is set as www.benkarpinski.com/landscapes. He has requested that we update it to http://bklandscapes.com/. His image is included in the Fall 2019 batch of new wallpapers.
```

![7997](https://user-images.githubusercontent.com/28145373/77434878-e6fcee80-6db7-11ea-9064-130c4011c9b7.png)

## Test plan
1. Load picture (thumbnail shown above)
2. Make sure image is attributed properly to Ben Karpinski

**Comment by LaurenWags** [comment]
cc @rebron as this was originally slated for 1.7.x

**Comment by bsclifton** [comment]
Problem found - original PR didn't update URL
https://github.com/brave/brave-core/pull/4398

Will submit a fix here shortly

**Comment by bsclifton** [comment]
Good catch, @LaurenWags 😄 Fixed in 1.7.x with https://github.com/brave/brave-core/commit/c6c8da70892b2d4610b93ee2d4435a9e6c2914f4 - should be in the next build we get

**Comment by LaurenWags** [comment]
Verified passed with

Brave | 1.7.73 Chromium: 80.0.3987.149 (Official Build) dev (64-bit)
-- | --
Revision | 5f4eb224680e5d7dca88504586e9fd951840cac6-refs/branch-heads/3987_137@{#16}
OS | macOS Version 10.14.6 (Build 18G3020)

* Verified STR from description
* Confirmed after getting the NTP image from thumbnail and clicking on the photographer's name I am directed to https://bklandscapes.com/
<img width="247" alt="Screen Shot 2020-03-25 at 3 54 20 PM" src="https://user-images.githubusercontent.com/28145373/77579629-0461b380-6eb1-11ea-8b27-9fbab91d9957.png">
* Clicked on a few other photographer names and confirmed I was directed to their sites
